test/unit/sandbox-libraries/pm.test.js

```diff
@@ -497,6 +505,82 @@ describe('sandbox library - pm api', function () {
                 assert.strictEqual(pm.variables.values.members[0].key, 'var1');
             `, {context: sampleContextData}, done);
         });
+
+        it('should get from collectionVariables', function (done) {
+            var contextData = {
+                collectionVariables: [{
+                    key: 'var1',
+                    value: 1,
+                    type: 'number'
+                }, {
+                    key: 'var2',
+                    value: 'two'
+                }]
+            };
+
+            context.execute(`
+                var assert = require('assert');
+
+                assert.strictEqual(pm.variables.get('var1'), 1);
+                assert.strictEqual(pm.variables.get('var2'), 'two');
+
+                // can set variables with same keys as collectionVariables
+                assert.strictEqual(pm.variables.get('var3', undefined));
+                assert.strictEqual(pm.variables.set('var3', 'hola'));
```

Comment:
?

Reply:
I might have accidentally removed `var3` from context, adding it back now. Good spot.